docs/serving/samples/knative-routing-go/README.md

```diff
@@ -39,26 +48,33 @@ cd $GOPATH/src/github.com/knative/docs
 
 2. Set your preferred container registry:
 
+If you use Google Container Registry (GCR). You will need a
```

Comment:
"You will need to enable the GCR API in your GCP project "

Reply:
Done. thx.